Source/RunActivity/Viewer3D/RollingStock/MSTSLocomotiveViewer.cs

```diff
@@ -2189,6 +2189,8 @@ public string GetControlName()
             return (Locomotive as MSTSLocomotive).TrainControlSystem.GetDisplayString(GetControlType().ToString());
         }
 
+        public string ControlLabel { get { return Control.Label; } }
```

Comment:
```suggestion
        public string ControlLabel => Control.Label;
```
You can also write it like that when you only have a getter.